examples/time_frequency/plot_time_frequency_erds.py

```diff
@@ -111,3 +113,85 @@
     fig.suptitle("ERDS ({})".format(event))
     fig.show()
+
+##############################################################################
+# Similar as for :class:`~mne.Epochs` objects, we can also export data from
+# :class:`~mne.time_frequency.EpochsTFR` and
+# `~mne.time_frequency.AverageTFR` objects to a
+# :class:`Pandas DataFrame <pandas.DataFrame>`:
+
+df = tfr.to_data_frame()
+df.time = df.time / 1000  # time to s
+df.head()
+
+##############################################################################
+# This allows us to use additional plotting functions like
+# :func:`seaborn.lineplot` to easily plot confidence bands:
+
+freq_bands = {'delta': (0.5, 4),
+              'theta': (5, 7),
+              'alpha': (8, 14),
+              'beta': (15, 35)}
+fig, axes = plt.subplots(4, 3, sharey=True, sharex=True)
+for f_idx, f_band in enumerate(['beta', 'alpha', 'theta', 'delta']):
+    data = df.loc[(df.freq >= freq_bands[f_band][0]) &
+                  (df.freq <= freq_bands[f_band][1])]
+    for ch_idx, ch in enumerate(["C3", "Cz", "C4"]):
+        g = sns.lineplot(data=data.reset_index(),
+                         x='time', y=ch, hue='condition',
+                         n_boot=10,  # only few repetitions for speed
+                         ax=axes[f_idx, ch_idx])
+        if f_idx == 0:
+            g.set_title(ch)
+        g.axhline(0, color='black', linestyle='dashed', linewidth=0.5,
+                  alpha=0.5)
+        g.axvline(0, color='black', linestyle='dashed', linewidth=0.5,
+                  alpha=0.5)
+        g.set(ylim=(-0.5, 1.5))
+        g.set_ylabel(f_band + '\n(%)', rotation=0, labelpad=20)
+        g.set_xlabel('Time (s)')
+        g.get_legend().remove()
+fig.subplots_adjust(top=0.9, left=0.1, right=0.9, bottom=0.2)
+fig.align_ylabels()
+handles, labels = g.get_legend_handles_labels()
+plt.figlegend(handles=reversed(handles), labels=reversed(labels),
+              loc='lower center', bbox_to_anchor=(0.1, 0.01, 0.9, 0.01),
+              ncol=2, title=None)
+plt.show()
+
+
+##############################################################################
+# Having the data in form of a DataFrame also facilitates subsetting,
+# grouping, and other transforms.
+# Here, we use seaborn to plot average ERDS in the motor-imaginery interval
+# as a function of frequency band and imagery condition:
+
+fbands_p = [((df['freq'] >= freq_bands[f][0]) &
+             (df['freq'] <= freq_bands[f][1])) for f in freq_bands]
+df['freq_band'] = np.select(fbands_p, freq_bands.keys())
+dat = (df.query('time > 1')
+       .filter(regex=r'condition|epoch|C3|Cz|C4|freq_band')
+       .groupby(['condition', 'epoch', 'freq_band'])
+       .mean()
+       .reset_index()
+       .melt(id_vars=['condition', 'epoch', 'freq_band'],
+             var_name='channel',
+             value_name='mean power'))
+fig, axes = plt.subplots(1, 2, sharey=True)
+for cond, ax in zip(['hands', 'feet'], axes.flatten()):
```

Comment:
here also I think you can avoid the for-loop by using `sns.FacetGrid` (no need to specify `row` this time though)

Reply:
Thanks! You're right. Also implemented.